engine/src/main/java/org/camunda/bpm/engine/impl/cfg/ProcessEngineConfigurationImpl.java

```diff
@@ -865,7 +865,8 @@
 
 
   // telemetry ///////////////////////////////////////////////////////
-  protected boolean telemetryEnabled = false;
+  /** if set to true the telemetry will be enabled from the first engine start*/
```

Comment:
The Javadoc should be a bit more verbose to make it more clear what the configuration does. E.g. explain that this will initialize the telemetry configuration only once when it has never been enabled/disabled before. After that, it can only be changed via API.

Reply:
I extended the javadoc, could you please have a look